packages/react-components/react-progress/src/components/Progress/useProgressStyles.ts

```diff
@@ -1,33 +1,356 @@
-import { makeStyles, mergeClasses } from '@griffel/react';
-import type { ProgressSlots, ProgressState } from './Progress.types';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+import { tokens, typographyStyles } from '@fluentui/react-theme';
+import { useFluent_unstable as useFluent } from '@fluentui/react-shared-contexts';
+import type { ProgressState, ProgressSlots } from './Progress.types';
 import type { SlotClassNames } from '@fluentui/react-utilities';
 
 export const progressClassNames: SlotClassNames<ProgressSlots> = {
   root: 'fui-Progress',
-  // TODO: add class names for all slots on ProgressSlots.
-  // Should be of the form `<slotName>: 'fui-Progress__<slotName>`
+  indicator: 'fui-Progress__indicator',
+  label: 'fui-Progress__label',
+  description: 'fui-Progress__description',
+};
+
+// Internal variables
+const progressBarColorPrimary = tokens.colorBrandStroke1;
```

Comment:
I think about this approach often, but it feels like a slippery slope.  Should we declare constants for all the widths too? IMHO, I think it is probably better to reference the tokens directly in makeStyles.